tests/test_metrics.py

```diff
@@ -71,47 +96,47 @@ def test_regional_rmse_values(
         to a reasonable (1e-7) precision."""
         output = np.array(
```

Comment:
I would strongly recommend saving this and other static assets in a .npz file and implementing a utility to load the data.